src/Microsoft.Azure.WebJobs.Logging/Entities/RecentPerFuncEntity.cs

```diff
@@ -108,20 +114,23 @@ public string FunctionInstanceId
         public string FunctionName { get; set; }
 
         public string DisplayName { get; set; }
-
-        public string RawStatus { get; set;  }
-
-        FunctionInstanceStatus IFunctionInstanceBaseEntry.Status
+        
+        public bool HasError
         {
             get
             {
-                FunctionInstanceStatus e;
-                if (!Enum.TryParse<FunctionInstanceStatus>(this.RawStatus, out e))
-                {
-                    return FunctionInstanceStatus.Unknown;
-                }
-                return e;
+                // Use RawStatus for legacy check. 
+                return _hasError || (this.RawStatus == "CompletedFailed");
+            }
+            set
+            {
+                _hasError = value;
             }
         }
+
+        /// <summary>
+        ///  Legacy for errors. We only read from this to check if it's "CompletedFailed" so we pick up legacy errors. 
```

Comment:
This legacy you're referring to is because the enum you're now using has name CompletedFailure instead of CompletedFailed?